src/slack/handlers/actions/checkin.ts

```diff
@@ -11,23 +13,54 @@ export const handleAppMentioned: EventMiddleware<'app_mention'> = async ({ event
             name: 'stopwatch'
         })
         const user = await client.auth.test()
-        const managers = await prisma.member.findMany({ where: { MemberCerts: { some: { Cert: { isManager: true } } } } })
-        const copres_string = config.slack.users.copres.join(',')
-        for (const manager of managers) {
-            if (!manager.slack_id) {
-                logger.warn('No slack id for manager ' + manager.email)
+
+        const departments = await prisma.department.findMany({
+            select: {
+                name: true,
+                Certs: {
+                    where: {
+                        isManager: true
+                    },
+                    select: {
+                        Instances: {
+                            select: {
+                                Member: {
+                                    select: {
+                                        email: true,
+                                        slack_id: true
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        })
+
+        const dept_managers = departments.map((dept) => ({
+            name: dept.name,
+            managers: dept.Certs.flatMap((cert) => cert.Instances.map((instance) => instance.Member.slack_id).filter((v) => v != null))
```

Comment:
```suggestion
            managers: dept.Certs.flatMap((cert) => cert.Instances.map((instance) => instance.Member.slack_id).filter((v) => v))
```

Reply:
Why? I'm not looking for falsey, I'm looking for null